MAINTAINERS.md

```diff
@@ -1,10 +1,34 @@
 # Maintainer instructions
 
+## Updating the tf provider
+1. modify go.mod to update the provider
+
+2. run `go get github.com/MaterializeInc/pulumi-frontegg/provider`
```

Comment:
I think you want to add a step to run `go mod tidy` here to bring go.sum into alignment.